src/app-layout/mobile-toolbar/index.tsx

```diff
@@ -141,7 +141,7 @@ export function MobileToolbar({
                 iconSvg={item.trigger.iconSvg}
                 badge={item.badge}
                 ariaLabel={item.ariaLabels?.triggerButton}
-                ariaExpanded={drawers.activeDrawerId !== undefined}
+                ariaExpanded={drawers.activeDrawerId === item.id}
```

Comment:
no `aria-controls` here?

Reply:
My thinking is that it isn't necessary on mobile, since you won't ever see the panel and the trigger button at the same time (its a full-screen overlay). So it would indicate that its controlling something that isn't in the DOM yet.